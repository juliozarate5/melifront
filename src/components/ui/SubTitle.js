import React from 'react';
import PropTypes from 'prop-types';

export default function SubTitle({subtitle}) {
    return (
        <div className="row my-4">
            <div className="col-lg-12 text-center">
                <h1>
                    {subtitle}
                </h1>
            </div>
        </div>
    )
}

SubTitle.propTypes = {
    subtitle: PropTypes.string.isRequired
}
