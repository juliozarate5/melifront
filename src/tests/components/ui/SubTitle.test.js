import React from "react";
import { shallow } from "enzyme";
import SubTitle from "../../../components/ui/SubTitle";

describe('Subtitle tests', () => {

    const subtitle = "subtitle";
    const wrapper = shallow(<SubTitle subtitle={subtitle}/>);

    test('showing component successful', () => {

        expect(wrapper).toMatchSnapshot()
    });

    test('must have h1 with subtitle', () => {
        const h = wrapper.find('h1');
        expect( h.text().trim() ).toBe(subtitle);

    })

})