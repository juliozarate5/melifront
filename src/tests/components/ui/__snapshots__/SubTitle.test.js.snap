// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle tests showing component successful 1`] = `
<div
  className="row my-4"
>
  <div
    className="col-lg-12 text-center"
  >
    <h1>
      subtitle
    </h1>
  </div>
</div>
`;
